Introduction.md

```diff
@@ -468,7 +468,11 @@ WebKit uses two primary management strategies when objects in other cases:
 
 ## Garbage collection in WebKit
 
-FIXME: Write this.
+See these blog posts:
+
+* [Understanding Garbage Collection in JavaScriptCore From Scratch](https://webkit.org/blog/12967/understanding-gc-in-jsc-from-scratch/)
+* [Introducing Riptide: WebKit’s Retreating Wavefront Concurrent Garbage Collector](https://webkit.org/blog/7122/introducing-riptide-webkits-retreating-wavefront-concurrent-garbage-collector/)
```

Comment:
I think it is helpful if we describe the brief summary of each blog post here, for example,

> Understanding Garbage Collection in JavaScriptCore From Scratch: Summary of how our GC is organized and how it works.
> Introducing Riptide: WebKit’s Retreating Wavefront Concurrent Garbage Collector: Focusing on novel algorithms of our GC including how to tackle web browser specific problems (managing DOM objects, concurrency, etc.).

Reply:
Make sense. I addressed it. https://github.com/WebKit/WebKit/pull/3754/commits/864f2f7cf88bca011d9cd205e08c46c6ca96ddf7